var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -88,6 +90,9 @@ class Openblas(MakefilePackage):
     provides("lapack@3.9.1:", when="@0.3.15:")
     provides("lapack@3.7.0", when="@0.2.20")
 
+    # https://github.com/xianyi/OpenBLAS/pull/2519/files
```

Comment:
It's great that you linked this, but does this mean that only certain versions need to apply this patch (e.g. it looks like that PR was merged in 2020, so perhaps all versions after that don't need this patch)?

Reply:
Decent point! I can try building after my next commit here and let you know